### ISSUE TITLE
undefined symbols after import root_numpy

### ISSUE THREAD

**Issue by Aelphy** [rootpy/root_numpy#272]
The problem saturated after using gcc 4.9 instead of 5.4

**Comment by kratsg** [comment]
@Aelphy -- have you tried re-installing `root_numpy` after reinstalling ROOT and numpy? It sounds like a linker error.

**Comment by Aelphy** [comment]
@kratsg -- Yes. I have tried to compile two versions of ROOT. Tried different versions of numpy.
The case was I could import ROOT library into python but while importing root_numpy the error about undefined symbols occurred.
Also I have tried different versions of root_numpy. Tried to install it using pip and to setup it manually.
The only case when the problem saturated was to change gcc from version 5 to the version 4.9
